<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>950</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>50</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>199</green>
          <blue>231</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>199</green>
          <blue>231</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>167</red>
          <green>199</green>
          <blue>231</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>soul_sync_logo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>250</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: #2C3E50;
}</string>
    </property>
    <property name="text">
     <string>Soul Sync</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>320</y>
      <width>731</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="error_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>731</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>389</y>
      <width>1180</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>60</y>
        <width>881</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    background-color: rgba(214, 230, 242, 0.85);
    border: 1px solid #4CAF50;
    border-radius: 8px;
    padding: 10px;
    margin-top: 20px;
    box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.1);
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="question_no_label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>231</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #2C3E50;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../../Downloads/src/view/UI files/Downloads/src/model/_c5dca270-092d-4483-b12e-a031080d7df9.jpeg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="question_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>851</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #2C3E50;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="back_btn">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>260</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 14px;
    font-weight: bold;
    border: none;
    border-radius: 12px; 
    padding: 10px 20px;
    min-width: 100px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}


QPushButton:hover {
    background-color: #5A9BD6;
}


QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
       </property>
       <property name="text">
        <string>Prev.</string>
       </property>
      </widget>
      <widget class="QPushButton" name="next_btn">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>260</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 14px;
    font-weight: bold;
    border: none;
    border-radius: 12px; 
    padding: 10px 20px;
    min-width: 100px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}


QPushButton:hover {
    background-color: #5A9BD6;
}


QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_btn1">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>160</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_btn2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>160</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_btn3">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>160</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_btn4">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>160</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_btn5">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>160</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>200</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #2C3E50;
}</string>
       </property>
       <property name="text">
        <string>Strongly Disagree</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>200</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #2C3E50;
}</string>
       </property>
       <property name="text">
        <string>Disagree</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>200</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #2C3E50;
}</string>
       </property>
       <property name="text">
        <string>Neutral</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>200</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #2C3E50;
}</string>
       </property>
       <property name="text">
        <string>Agree</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>200</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #2C3E50;
}</string>
       </property>
       <property name="text">
        <string>Strongly Agree</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>30</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
    font-size: 15px;
    font-weight: bold;
    color: #2C3E50;
    padding: 10px;
    border-bottom: 2px solid #5A9BD6;
    text-align: center;
}
</string>
      </property>
      <property name="text">
       <string>Let's Assess Your Stress</string>
      </property>
     </widget>
     <widget class="QPushButton" name="results_btn">
      <property name="geometry">
       <rect>
        <x>519</x>
        <y>440</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 14px;
    font-weight: bold;
    border: none;
    border-radius: 12px; 
    padding: 10px 20px;
    min-width: 100px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}


QPushButton:hover {
    background-color: #5A9BD6;
}


QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
      </property>
      <property name="text">
       <string>Results</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="message_label">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>70</y>
        <width>1081</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{color: #2C3E50;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="main_btn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 14px;
    font-weight: bold;
    border: none;
    border-radius: 12px; 
    padding: 10px 20px;
    min-width: 100px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}


QPushButton:hover {
    background-color: #5A9BD6;
}


QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
